htdocs/comm/action/index.php

```diff
@@ -528,7 +528,7 @@
 if ($user->rights->agenda->myactions->create || $user->rights->agenda->allactions->create) {
 	$tmpforcreatebutton = dol_getdate(dol_now(), true);
 
-	$newparam .= '&month='.str_pad($month, 2, "0", STR_PAD_LEFT).'&year='.$tmpforcreatebutton['year'];
+	$newparam = 'month='.str_pad($month, 2, "0", STR_PAD_LEFT).'&year='.$tmpforcreatebutton['year'];
```

Comment:
You need to keep &month because is put after &backtopage

Reply:
not when the variable is undefined, but now both cases are covered :)